example/src/main/java/org/apache/rocketmq/example/ordermessage/Consumer.java

```diff
@@ -28,12 +28,19 @@
 
 public class Consumer {
 
+    public static final String CONSUMER_GROUP = "please_rename_unique_group_name_3";
+    public static final String DEFAULT_NAMESRVADDR = "127.0.0.1:9876";
+    public static final String TOPIC = "TopicTest";
+    public static final String SUB_EXPRESSION = "TagA || TagC || TagD";
+
     public static void main(String[] args) throws MQClientException {
-        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer("please_rename_unique_group_name_3");
+        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer(CONSUMER_GROUP);
 
+        // If the debugging source code can open comments, you need to set the namesrvAddr to the address of the local namesrvAddr
+//        consumer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```

Comment:
```suggestion
      //  consumer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```
My obsessive compulsive disorder